server/server.js

```diff
@@ -245,13 +282,19 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
       (this.subs[to] ? 'is subscribed' : 'not subscribed')
     );
 
-    this._persistClientData(socket, message);
-    this._storeResource(resource);
-    this._persistenceSubscribe(resource.to, socket.id);
-    this._updateLimits(socket, message, resource.options);
-    this._stampMessage(socket, message);
-    resource.handleMessage(socket, message);
-    this.emit(message.op, socket, message);
+    this._runMiddleware('post', socket, message, messageType, function (err) {
```

Comment:
There is. We need that part for when we make the rate limiter into a middleware. In this particular area, we need to do `this._updateLimits(socket, message, resource.options)`, which is the post hook.